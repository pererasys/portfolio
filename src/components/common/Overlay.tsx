/**
 * @author Andrew Perera
 * Copyright (C) 2021
 */

import React, { DetailedHTMLProps, HTMLAttributes } from "react";
import styles from "../../styles/components/common/Overlay.module.scss";

const Overlay: React.FC<
  DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>
> = (props) => <div className={styles.overlay} {...props}></div>;

export default Overlay;
