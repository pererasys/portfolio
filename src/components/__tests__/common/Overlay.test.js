import { create } from "react-test-renderer";

import Overlay from "../../common/Overlay";

describe("<Overlay />", () => {
  const setup = () => {
    const component = create(<Overlay />);

    return {
      component,
    };
  };

  it("should render without crashing", () => {
    const { component } = setup();

    expect(component.toJSON()).toMatchSnapshot();
  });
});
