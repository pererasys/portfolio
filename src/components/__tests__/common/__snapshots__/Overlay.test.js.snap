// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Overlay /> should render without crashing 1`] = `<div />`;
